src/main/java/com/epam/mjc/collections/list/ArrayListCreator.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.mjc.collections.list;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayListCreator {
+    public ArrayList<String> createString(List<String> sourceList) {
```

Comment:
Could you please rename this method? `createString` can't return `ArrayList<String>`